numpy/_core/tests/test_cython.py

```
@@ -53,7 +53,12 @@ def install_temp(tmpdir_factory):
         subprocess.check_call(["meson", "setup", str(srcdir)],
                               cwd=build_dir
                               )
-    subprocess.check_call(["meson", "compile", "-vv"], cwd=build_dir)
+    try:
+        subprocess.check_call(["meson", "compile", "-vv"], cwd=build_dir)
+    except subprocess.CalledProcessError as p:
+        print(f"{p.stdout=}")
+        print(f"{p.stderr=}")
+        raise
```

Try to print out the error instead of swallowing it